includes/admin/stripe-settings.php

```diff
@@ -3,6 +3,33 @@
 	exit;
 }
 
+if ( wc_stripe()->connect->is_connected() ) {
+	$reset_link = add_query_arg(
+		array( 'reset_stripe_api_credentials' => true ),
+		admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe' )
+	);
+
+	$api_credentials_text = sprintf(
+		__( '%1$sReset Stripe account keys%2$s', 'woocommerce-gateway-stripe' ),
```

Comment:
Let's make it clearer what this will cause to happen. Could we change the linked text to

Clear all Stripe account keys. This will disable any connection to Stripe.

Reply:
Sure thing! Changed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1267/commits/f9ad9f38bba43311f5d8998705e2b71325202034